### PR TITLE
Yda 5575 fileviewer

### PR DESCRIPTION
**Pull request by claravox**

### CODE REVIEW

File: vault.py [hunk 1]
```diff
@@ -607,6 +609,7 @@ def api_vault_collection_details(ctx, path):
         "basename": basename,
         "status": status,
         "metadata": metadata,
+        "member_type": member_type,
```
Comment (claravox) [inline vault.py:619]: Do we have api versions? Would adding this break anything?

File: browse.py [hunk 2]
```diff
@@ -315,3 +317,46 @@ def _filter_vault_deposit_index(row):
...
+
+    # Check end extension
+    valid_extension = False
+    for ext in config.text_file_extensions:
+        ending = '.' + ext
+        if file_path.lower().endswith(ending):
+            valid_extension = True
+
+    if not valid_extension:
+        return api.Error('not_valid', 'The given data object does not have a valid file extension')
```
Comment (claravox) [inline browse.py:349]: I'm not sure how to pick names for the api errors.

Comment (lwesterhof) [reply browse.py:349]: I think you did great :)

File: vault.py (hunk 1 above)
Comment (lwesterhof) [reply vault.py:619]: API version is in sync with the release version and can be found here: https://github.com/UtrechtUniversity/yoda-ruleset/blob/development/__init__.py#L4

Could be, but I don't think it does in this case.

**Comment by lwesterhof** [comment]
Screenshot_2024-02-14_08-47-18

**Review by lwesterhof — APPROVED** [review]
LGTM!

**Pull request merged**